src/components/data-management/UploadDetailsModal.jsx

```diff
@@ -25,30 +25,27 @@ const UploadDetailsModal = (props) => {
   } = props;
   const { fileCategory, sampleUuid } = uploadDetailsModalDataRef.current || false;
   const file = uploadDetailsModalDataRef.current?.file || {};
-  const {
-    upload = {}, bundle = {},
-  } = file;
+  const { upload } = file || {};
```

Comment:
Use `??` instead for null checks

Reply:
also perhaps do this same change to 
```
const { fileCategory, sampleUuid } = uploadDetailsModalDataRef.current || false;
  const file = uploadDetailsModalDataRef.current?.file || {};
```

Reply:
Also if you can change the usage of false to null instead in `uploadDetailsModalDataRef.current || false`

Reply:
changed